### ISSUE TITLE
gnt-network info outputs wrong external reservations (addresses offset by +30)

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1124]
Originally reported of Google Code with ID 1068.

```
Using ganeti 2.12.0 with a /24 network, gnt-network outputs wrong externally reserved addresses:

  Subnet: 10.44.2.0/24
  externally reserved IPs:
    10.44.2.30, 10.44.2.31, 10.44.2.32, 10.44.2.33, 10.44.2.72,
    10.44.3.29

For a start, 10.44.3.29 is not even part of the subnet. Looking at config.data, the bitmap is correct, reserving network, router, nodes and broadcast IPs:

"ext_reservations": "1111000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001"

It looks as if the reported IP addresses are offset by 30. The same happens with the other network we defined, a /28 where all reported IPs are offset by 30 as well.
```
Originally added on 2015-04-17 09:03:03 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-04-17 09:37:12 +0000 UTC.

Added Labels: Priority-Medium
Changed State: Started

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in

commit 2a050e1939d1f576e5008b69deebd9c9e4e3c086
Author: Petr Pudlak <pudlak@google.com>
Date:   Fri Apr 17 19:06:19 2015 +0200

    Fix the computation of the list of reserved IP addresses
    
    After refactoring the code for network addresses to Haskell, a
    regression appeared that added 30 to all IP addresses in the generated
    list. Since the code for generating the list used just 'show' and look
    for 0s and 1s in the string (and treating anything else as 0), it turned
    out that after adding a separate type for address pools, the strings
    become
    
        "AddressPool {apReservations = ...}"
    
    and the initial part of the string was exactly the mysterious 30
    characters.
    
    This fix replaces this by switching to correct types and iterating over
    the bits in BitArray.
    
    Fixes #1068.
    
    Signed-off-by: Petr Pudlak <pudlak@google.com>
    Reviewed-by: Klaus Aehlig <aehlig@google.com>

```
Originally added on 2015-04-17 17:24:14 +0000 UTC.

Added Labels: Usability Component-networking
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-04-29 10:54:44 +0000 UTC.

Changed State: Released
